nil/internal/telemetry/internal/resource.go

```diff
@@ -2,7 +2,7 @@ package internal
 
 import (
 	"go.opentelemetry.io/otel/sdk/resource"
-	semconv "go.opentelemetry.io/otel/semconv/v1.27.0"
+	semconv "go.opentelemetry.io/otel/semconv/v1.26.0"
```

Comment:
Let's add a comment on why it's important to use this particular version

Reply:
I'm not sure it's important. I hope it should be fixed on otelcol-contrib side. But I didn't find specific way in their documentation

Reply:
Ok, we could just leave a warning about possible issues with the change that are worth checking out)

Reply:
It's reasonable